add: nothing to see here

NightDawnFox opened the pull request:
<!-- Пишите **НИЖЕ** заголовков и **ВЫШЕ** комментариев, иначе ваш текст может не отобразиться. -->
<!-- В Contributing.MD вы можете найти некоторые рекомендации к оформлению пулл-реквеста. -->

## Описание
Мне пока нечего тут написать
<!-- Опишите, что делает ваш ПР. Документировать каждую деталь не требуется, просто укажите основные изменения. -->

## Ссылка на предложение/Причина создания ПР
У меня пока нет причин делать этот ПР
<!-- Здесь оставьте ссылку на сообщение в #отчеты-по-предложениям, чтобы подтвердить, что ваше предложение одобрено. Либо укажите, почему этот ПР должен пройти без предложки. -->
<!-- Пример ссылки: https://discord.com/channels/617003227182792704/755125334097133628/ID-сообщения -->

## Демонстрация изменений
Мне пока нечего вам показывать
<!-- Здесь вы можете показать изменения внешне, к примеру новые спрайты, звуки или изменения карты. Или написать, что именно изменилось для игроков. Этот пункт полностью опционален и его можно удалить. -->

NightDawnFox committed:
thats all!

NightDawnFox committed:
Revert "thats all!"

This reverts commit 947524612688087a560cf118f694d7a670a92b85.

NightDawnFox committed:
Merge remote-tracking branch 'upstream/master220' into master220

NightDawnFox committed:
Merge remote-tracking branch 'upstream/master220' into master220

NightDawnFox committed:
Merge remote-tracking branch 'upstream/master220' into master220

NightDawnFox committed:
Merge remote-tracking branch 'upstream/master220' into master220

NightDawnFox committed:
Merge remote-tracking branch 'upstream/master220' into master220

NightDawnFox committed:
Merge remote-tracking branch 'upstream/master220' into master220

NightDawnFox committed:
Merge remote-tracking branch 'upstream/master220' into master220

NightDawnFox committed:
Merge remote-tracking branch 'upstream/master220' into master220

NightDawnFox committed:
Merge remote-tracking branch 'upstream/master220' into master220

NightDawnFox committed:
Merge remote-tracking branch 'upstream/master220' into master220

NightDawnFox committed:
Merge remote-tracking branch 'upstream/master220' into master220

NightDawnFox committed:
Merge remote-tracking branch 'upstream/master220' into master220

NightDawnFox committed:
that's all!

NightDawnFox committed:
Revert "that's all!"

This reverts commit 5dc611263ee3e3ec64cdcddfb1b0aba1379083cd.

NightDawnFox committed:
Merge remote-tracking branch 'upstream/master220' into master220

NightDawnFox committed:
Merge remote-tracking branch 'upstream/master220' into master220

NightDawnFox committed:
Merge remote-tracking branch 'upstream/master220' into master220

NightDawnFox committed:
Merge remote-tracking branch 'upstream/master220' into master220

NightDawnFox committed:
Merge remote-tracking branch 'upstream/master220' into master220

NightDawnFox committed:
Merge remote-tracking branch 'upstream/master220' into master220

NightDawnFox committed:
Merge remote-tracking branch 'upstream/master220' into master220

NightDawnFox committed:
Merge remote-tracking branch 'upstream/master220' into master220

NightDawnFox committed:
Merge remote-tracking branch 'upstream/master220' into master220

NightDawnFox committed:
Merge remote-tracking branch 'upstream/master220' into master220

NightDawnFox committed:
Merge remote-tracking branch 'upstream/master220' into master220

NightDawnFox committed:
Merge remote-tracking branch 'upstream/master220' into master220

NightDawnFox committed:
Merge remote-tracking branch 'upstream/master220' into master220

NightDawnFox committed:
Merge remote-tracking branch 'upstream/master220' into master220

NightDawnFox committed:
Merge remote-tracking branch 'upstream/master220' into master220

NightDawnFox committed:
Merge remote-tracking branch 'upstream/master220' into master220

NightDawnFox committed:
1

NightDawnFox committed:
2

NightDawnFox committed:
Merge remote-tracking branch 'upstream/master220' into master220

NightDawnFox committed:
3

NightDawnFox committed:
new lavaruin - spaceship

NightDawnFox committed:
hotfix

NightDawnFox committed:
hotfix2

ROdenFL commented:
Зачем вообще что-то делать, если +5000 строчек это "нечего смотреть"?)

NightDawnFox committed:
господи блядь как я ненавижу ебаный мапинг убейте меня кто-нибудь

NightDawnFox commented:
@ROdenFL Потому что оно еще не готово!

NightDawnFox committed:
Update lavaland_surface_pirateship.dmm

NightDawnFox committed:
sprites

definitelynotspaghetti commented:
Прикольные амулеты. Я сам когда-то раздумывал над добавлением колец из дарк соулс на парадиз, но в итоге забил на эту идею

NightDawnFox committed:
sprites2

NightDawnFox committed:
yesss

NightDawnFox committed:
mapping 2

NightDawnFox committed:
Merge remote-tracking branch 'upstream/master220' into master220

NightDawnFox committed:
Закончил с украшениями и гемами.

У гемов теперь уникальные свойства, почти у каждого. Новые украшения, использующие эти самые гемы. В целом, красиво и прикольно.

NightDawnFox committed:
fix

я дебил

Mindfri committed:
Tweak: Toy Figures are Small now. (#3656)

Vladisvell committed:
tweak: changed cyborg's storage increase upgrade name (#3662)

Gottfrei committed:
refactor: Space Dragon Cleanup (#3657)

aeternaclose committed:
bugfix: Cyberiad Lavaland Outpost  (#3640)

Vladisvell committed:
bugfix: fixed incorrect icon rendering upon reloging into robots and creatures (#3651)

NightDawnFox committed:
bugfix: golem targeter fix and no more alien multitool on golem shuttle (#3646)

Vladisvell committed:
bugfix: removed repeating description for hacked crates. (#3650)

captainnelly committed:
Add: mosin nagant for cargonia (#3658)

Dimach committed:
Tweak: nuke missed station only if exploded on space turf (#3669)

LiquidPotroh committed:
add: Draconids & Ash Walkers Customization (#3661)

NightDawnFox committed:
bugfix: normal tail wagging for new unathi tails (#3664)

Etrnlmelancholy committed:
add: Chameleon meson glasses (#3634)

LoneAsket committed:
Add: Grab intent gag and blind function (#3568)

Vladisvell committed:
bugfix: fixed crossbow arrows damage scaling (#3663)

Vladisvell committed:
add: Colorful buckets and Cleanbots (#3597)

gluten5659 committed:
TWEAK: Buckle and paper show delay (#3587)

Daeberdir committed:
add: fabricators rotatability. (#3649)

ROdenFL committed:
add: Baked vulp (#3655)

Daeberdir committed:
tweak: balaclava icon change (#3666)

Vladisvell committed:
qol: Metal Foam platings can be replaced with normal one by using metal sheet (#3574)

ROdenFL committed:
TWEAK: hypospray recolour (#3659)

actions-user committed:
Automatic changelog generation

Dimach committed:
Revert: "Tweak: Change Sec HUD icons of Syndicate" (#3670)

NightDawnFox committed:
Новые голиафы, часть 1

Теперь голиафы рычат (через мишку), когда с ними дерешься. Теперь голиафы могут чарджиться на вас и таранить. Теперь вблизи голиафы испускают "защитную стену" из тентаклей. Пока всё, я хочу спать

ROdenFL commented:
Хорошая работа, файнали, голиафы убивают Кроноса в два раза чаще

NightDawnFox committed:
Новые голиафы, часть 2

Теперь голиафов можно "приручить" едой, делая их дружелюбными к тебе. Так же добавляет голиафам шанс на рефлект, который блокирует 30 (50 для древних)% выстрелов, отражая их в стороны и иногда в отправителя. На этом работа с голиафами закончена.

NightDawnFox committed:
Небольшой нерф/бафф наблюдателей

Теперь обычные наблюдатели не замараживают, а роняют куклу, накладывают 3 секунды замедления и конфуза, давая окно в пару миллисекунд перед новым выстрелом, а так же наносят 10 берна за выстрел. Теперь магма и айсвинги спавнятся гораздо чаще

NightDawnFox committed:
Сильный бафф нестандартных вотчеров

NightDawnFox committed:
Доработка Легионов

Теперь тела "игроков", выпавшие из легиона, имеют в себе опухоль, которая таит в себе отложенную смерть (если лазарет решит не клонировать тело, а лечить), превращающая его в легиона. Жууууть.

NightDawnFox committed:
Делаем легионов В РАЗЫ ОПАСНЕЕ

Сейчас легионы представляют из себя мальчиков для битья, не являясь какой-либо угрозой. Теперь же они ставят шахтеров под угрозу выживания. Если им не повезет + шахтерам - они превратятся в новых легионов будучи живыми.
В обмен на это - трупы с легионов падают идеально целыми. Только и дефибай..

NightDawnFox committed:
Увы

NightDawnFox committed:
увы2

унатхи теперь тоже под угрозой. Защищены только люди в ригах.
Умершие от "заражения" имеют шанс в 25% на то, чтобы залутать второй тумор

NightDawnFox committed:
Новый подтип мароу уивера - ледяной

Впрыскивает фростоил. Спрайты с йогов. Смешное

NightDawnFox committed:
Упс.

aeternaclose committed:
add: MultiId (#3674)

aeternaclose committed:
bugfix: NoDuplicatesForXenos (#3688)

Rerik007 committed:
Feat: changeable colors for underclothing (#3256)

Rerik007 committed:
bugfix: second slowed cancel first (#3672)

captainnelly committed:
Add: standart first aid kit for crew (#3684)

Vladisvell committed:
bugfix: fixed buckets color on head (#3689)

Vladisvell committed:
bugfix: Preferences (and voice seed) now transfer correctly to AI on roundstart. (#3679)

NightDawnFox committed:
bugfix: bubblegum now charge hallucinations properly (#3675)

PlayTheGoat committed:
tweak: change sec HUD icon for deathsquad (#3671)

captainnelly committed:
balance: flashbangs (#3677)

Vladisvell committed:
add: Bombarda, handmade grenade launcher. (#3472)

Gottfrei committed:
refactor: Separate Malf AI Datum (#3612)

Gottfrei committed:
bugfix: Changeling Rare Bug Fix (#3681)

captainnelly committed:
balance: energy katana is now harmfull to non ninja (#3683)

Vladisvell committed:
balance: ninjas are no longer affectable by pepper sprays (#3691)

Vladisvell committed:
bugfix: guest_ban entry value (#3697)

SAADf603 committed:
bugfix: Delta library and some more (#3668)

captainnelly committed:
balance: adrenaline implant (#3676)

Daeberdir committed:
bugfix: sprite corrections by @piotrthetchaikowsky (#3692)

actions-user committed:
Automatic changelog generation

NightDawnFox committed:
Рыбалка

Теперь миазмы создают в себе хранилище, если туда попадает человек. Теперь на всех аванпостах есть удочка. Теперь шахтеров можно вылавливать из миазм.

NightDawnFox committed:
Баффы ручных шахтерских инструментов

Ну, всё сказано в названии

NightDawnFox committed:
Update lavaland_surface_seed_vault.dmm

NightDawnFox committed:
ой.

NightDawnFox committed:
ой-ёй.

NightDawnFox committed:
Плюс декали, деревянные полы, грязь и мусор.

NightDawnFox committed:
ой...

NightDawnFox committed:
fuck

NightDawnFox committed:
Набор багфиксов и балансных правок

1. У перчаток голиафа больше защиты
2. Костюм первопроходца теперь имеет такие же слоты, как и накидка голиафа
3. Усиленная кровь дракона теперь имеет шанс в 1/5, чем 1/16
4. Шаманы пеплоходцев теперь не роняют гарантированно
5. Проджектайлы ватчеров медленные
6. Лук теперь только на спину, добавлены спрайты
7. В колчан теперь лезет 21 стрела, а не 14 (ранее должно было 20, хдд)
Потом нужно фиксить маппинг, ыааа

NightDawnFox committed:
балансные правки 2

Режем дроп у уиверов

NightDawnFox committed:
Мне нельзя доверять баланс.

WolfLox committed:
tweak: crewmember-aid-kit (#3726)

ROdenFL committed:
tweak: wallets can hold patches (#3719)

NightDawnFox committed:
bugfix: normal drake hood for vulps (#3721)

ROdenFL committed:
fix: pistol magazine return his sprite (#3702)

Mindfri committed:
add: IPC Combat Upgrade in uplink (#3704)

Vladisvell committed:
tweak: Bombarda tweaks (#3699)

Daeberdir committed:
bugfix: small map fixes. (#3711)

LoneAsket committed:
FIX: Allow Skeleton Race plays on his Bones (#3709)

ROdenFL committed:
bugfix: wolpin with vulps head (#3707)

Daeberdir committed:
bugfix: wrong check for martial arts usability. (#3694)

Daeberdir committed:
bugfix: nucleation organs desc typo. (#3698)

Vladisvell committed:
add: brand new items for Thunderdome and bundles (#3722)

p4el007 committed:
add:sexy mustache beards and hairstyles (#3682)

NightDawnFox committed:
add: mirror on beach biodome (#3706)

Vladisvell committed:
bugfix: Now blob can't expand into transitional borders of the Z-level (#3714)

Daeberdir committed:
bugfix: separated objectives for different weapons. (#3708)

PlayTheGoat committed:
qol: fixing inventory cells for silicons (#3700)

Vladisvell committed:
bugfix: fixed incorrect improvised shotgun rendering (#3724)

SAADf603 committed:
Fix: old Z2 emo shuttles (#3687)

Mindfri committed:
tweak: BattlemageRecharges 3 FOR 1 (#3718)

Gottfrei committed:
tweak: Vampire Tweaks and Fixes (#3660)

NightDawnFox committed:
add: eye implants in loadout (#3368)

ROdenFL committed:
add: headslug event (#3577)

Etrnlmelancholy committed:
tweak: ERT equip tweaks (#3680)

Gottfrei committed:
bugfix: Spell Cooldown Visual (#3725)

actions-user committed:
Automatic changelog generation

NightDawnFox committed:
Полировка рыбалки

Теперь генокрады и реджувнутые куклы могут вылезать из чазма

NightDawnFox committed:
"реворк" колосса

Мне в лом расписывать, я просто портировал его атаки и кулдауны с оффов и ТГ. Потом расскажу, че я сделал

NightDawnFox committed:
Шахтерские рации (работают)

Порт с ТГ шахтерских погодных раций, заранее оповещающих шахтеров о приближающейся буре. Код несколько кривой, но работает.. За спрайты спасибо глав.спрайтеру

NightDawnFox committed:
map

NightDawnFox committed:
Кривые лапы

NightDawnFox committed:
да блядь.

NightDawnFox committed:
Пометка шпилей! Новые типы шпилей!

Теперь сканирование тендриллов с помощью сканера руды позволяет их "отмечать" на ГПС и отслеживать их на z-уровне. Добавлено два типа новых шпилей, спавнящие фауну рандомно и чаще. Повышена частота спавна шпилей. Изменен mob_spawn_list Для большего удобства.

NightDawnFox committed:
огненный цветок

Порт нового растения с ТГ/шиптеста - fire blossom. Светится в темноте, при сборе дает светящиеся цветки. Красивые, можно носить на голове

Honker9999999 committed:
fix: no stimulants for nukies (#3744)

Gottfrei committed:
bugfix: Get Temperature Fix (#3732)

Vladisvell committed:
bugfix: now random item picking from uplink is working correctly (#3731)

WolfLox committed:
fix: wrong-mantisblades-sprite (#3741)

captainnelly committed:
soundadd: revolvers (#3740)

Daeberdir committed:
tweak: MA combo steps explain_text. (#3734)

Daeberdir committed:
bugfix: ticket machine button works (#3735)

ArtemDerp committed:
bugfix: lavaland clown dome set lava base turf (#3736)

WolfLox committed:
add: frag grenade for traitors (#3727)

Gottfrei committed:
tweak: Shadowling Glare Autotarget (#3730)

actions-user committed:
Automatic changelog generation

Gottfrei committed:
bugfix: Malf AI Destroy Datum Fix (#3752)

NightDawnFox committed:
Порт режима "HRD-MDE" с оффов

Гигантская работа товарища "Qwertytoforty" по усложению лавы ради медалек теперь будет и у нас (возможно). Нужно проверить и протестить, но у меня уже позднее время.

NightDawnFox committed:
fix 1

NightDawnFox committed:
Казалось бы, одна лишняя скобка

paradise.mdme.dmb - 9772 errors, 1554 warnings (10/22/23 5:21 pm)

NightDawnFox committed:
фиксы маппинга

NightDawnFox committed:
Оверрайдер подрывных зарядов

Превращает безопасные шахтерские подрывные заряды в полноценную взрывчатку. Предмет синдиката. Цифры - позже.

NightDawnFox committed:
Набор портов с оффов и инфляции

1. Добавлен джаунтер-граната, создающая джаунтеры для всех точек на карте и являющаяся альтернативным способом доставать шахтеров с миазм
2. Порт с ТГ посоха легиона
3. В вендоры добавлен шприц с углем
4. Во всех вендорах резко выросла цена на всё.
5. Цена на харднесс стала ниже, однако было добавлено ограничение на один модуль
6. toxix -> toxin в описании инъектора угля
7. На все лавовые карты был добавлен маркер ГПС, ведущий на аванпост
8. Со всех карт были удалены ГПС, они были перемещены в хранилища.

ROdenFL commented:
Последний коммит прям нравитца

NightDawnFox committed:
Revert: seed vault + shelter capsules tweak

Не хочу ебать себе мозги с прохождением маппинг-предложек..
Более нельзя открывать капсулы на Z-уровне станции, пока ты их не емагнешь.

NightDawnFox committed:
Я даже в реверте умудрился..

NightDawnFox committed:
Твики хардмода мегафауны

Порт с оффов багфиксов, ничего особенного

NightDawnFox committed:
ТГУИ.

Теперь карты отслеживают количество общего количества добытых ресурсов

NightDawnFox committed:
чутка правим рантаймы и переносим карту

ДА.

NightDawnFox committed:
Фиксы багов

NightDawnFox committed:
Проверка для убирания рантаймов

"undefined proc or verb /obj/machinery/porta_turret/syndicate/can inject()

NightDawnFox committed:
Небольшие приколы пеплоходцам ч.1

1. Теперь убийство некрополя пеплов даёт нормальное ядро
2. Теперь у пеплоходцев, шамана и драконида есть встроенная зажигалка. Подробности потом
3. Теперь пеплоходцы теряют свой бег на станции. Увынск.
4. Шаман пеплоходцев теперь может чувствовать местонахождение мировой кузницы.

NightDawnFox committed:
Смешное добавление пеплоходцам

Теперь все магмитовые вещи могут использоваться пеплоходцами

NightDawnFox committed:
обновляем названия карты

NightDawnFox committed:
Починка атмоса

Никакого нормального давления на картах (возможно я всё сломал)

NightDawnFox committed:
Update gems.dm

NightDawnFox committed:
Revert "Update gems.dm"

This reverts commit 72d951ff6a28d1c35443f24d9835a1e1e8dfee1b.
